qoqo/tests/integration/operations/spin_boson_operations.rs

```diff
@@ -0,0 +1,670 @@
+// Copyright © 2021-2023 HQS Quantum Simulations GmbH. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+// in compliance with the License. You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software distributed under the
+// License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+// express or implied. See the License for the specific language governing permissions and
+// limitations under the License.
+
+use pyo3::prelude::*;
+use pyo3::Python;
+use qoqo::operations::convert_operation_to_pyobject;
+use qoqo::operations::{
+    QuantumRabi, LongitudinalCoupling, JaynesCummings
```

Comment:
I'm confused as to why this import works, these names should end in `Wrapper`

Reply:
It was probably not being run because the test file was not being imported in mod.rs. I have added it now and appended `Wrapper` to these imports